torch/csrc/jit/OVERVIEW.md

```diff
@@ -718,46 +721,46 @@ t2.relu_() # inplace relu operator, note t is modified as well!
 * requires_grad - whether the tensor is recording its gradient with autograd
 
-Changes in these properties change how operators on tensor will evaluate and would make certain optimization invalid. For instance, if we have fuser capable of generating new cuda kernels but not cpu kernels, it is only valid to fuse operations where the inputs are known to run only on CUDA devices. The GraphExecutor's job is to still enable optimization even when certains combinations of properties prevent optimizations from occurring.
+Changes in these properties change how operators on tensors will evaluate and would make certain optimization invalid. For instance, if we have fuser capable of generating new CUDA kernels but not CPU kernels, it is only valid to fuse operations where the inputs are known to run only on CUDA devices. The GraphExecutor's job is to still enable optimization even when certain combinations of properties prevent optimizations from occurring.
 
-Nodes in a graph are executed *serially* in the order they appear in a block. Nodes may be reordered either during optimization or by the interpreter itself if it can be proven that
-it is not distinguishable from the original serial execution order. These semantics are necessary since the combination of mutable tensors and potential alias between tensors makes it unsafe to perform arbitrary reordering otherwise. However, the AliasInfo object can accurately track how alias propagate through builtin operators so optimization passes can query when certain reorders or optimizations are safe.
+Nodes in a graph are executed *serially* in the order they appear in a block. Nodes may be reordered either during optimization or by the interpreter itself if it can be proven that the new order
+is not distinguishable from the original execution order. These semantics are necessary since the combination of mutable tensors and potential aliases between tensors makes it unsafe to perform arbitrary reordering otherwise. However, the AliasInfo object can accurately track how alias propagate through builtin operators so optimization passes can query when certain reorders or optimizations are safe.
 
-We also provide user-accessible parallel execution through the `fork` and `wait` primitives. The `fork` primitive begins execution of `fn` in parallel with the current thread of execution, immediately returning a Future object that will hold the result of the forked function. The `wait` method of the future then causes the invoking task to wait for the value being computed on the forked task.
+We also provide user-accessible parallel execution through the `fork` and `wait` primitives. The `fork` primitive begins execution of `fn` in parallel with the current thread of execution, immediately returning a Future object that will hold the result of the forked function. The `wait` method of the future then causes the invoking thread to wait for the value being computed on the forked thread.
```

Comment:
I was initially a bit confused because the sentence used both terms ("current thread of execution... invoking task..."). As far as I can tell the terms were being used to refer to a single concept and I found it confusing. After looking into the code a bit, it seems to me that "current thread" and "invoking thread" are both referring to the same thing, while the future may or may be implemented under the hood with a separate thread, so I just reworded it to avoid introducing a new term.